src/api/Feed.js

```diff
@@ -1768,7 +1764,6 @@ class Feed extends Base {
      * @param {BoxItem} file - The file to which the task is assigned
      * @param {string} commentId - Comment ID
      * @param {string} text - the comment text
-     * @param {boolean} hasMention - true if there is an @mention in the text
```

Comment:
Was this accidentally removed? It looks like we're still using this parameter in `updateComment` and several other methods.

Reply:
Yes, definitely. I need to bring it back.